### PR TITLE
Chore: fix readme

### PR DESCRIPTION
**Pull request by dduportal**
Following #152 , this PR correct the readme to mention the github issue tracker (instead of the old jira) + other fixes to help users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,10 +40,15 @@ Feel free to follow the next action points:
 * Generate your private key and certificate request: `./easyvpn request <your-jenkins-username>`
   Your private key will be generated in `./cert/pki/private`
 
-  > ### This key **must** remain **secret**!
+  > ⚠️ This key **must** remain **secret**! ⚠️
+
+* Open an issue on [jenkins-infra/helpdesk](https://github.com/jenkins-infra/helpdesk) describing the reason why you need an access to the VPN
+* Create a new pull request on [jenkins-infra/docker-openvpn](https://github.com/jenkins-infra/docker-openvpn)
```
Comment (lemeurherve) [inline README.md:46]:
```suggestion
* Create a new pull request on [jenkins-infra/docker-openvpn](https://github.com/jenkins-infra/docker-openvpn)
* Open an issue on [jenkins-infra/helpdesk](https://github.com/jenkins-infra/helpdesk) describing the reason why you need an access to the VPN, and referencing your pull request
```

**Review by lemeurherve — APPROVED** [review]
LGTM
Wondering if we should prepopulate new issue fields in the URL

File: README.md (hunk 1 above)
Comment (lemeurherve) [reply README.md:46]: Reverted the order so the issue can contains a link to the PR previously created

Comment (dduportal) [reply README.md:46]: Sorry to disagree, but an issue should be open first, and then amended.

Opening a PR before the associated issue feels really weird 😅

(my rule of thumb is "start with why" and "issue == why + what | PR == how")

Comment (lemeurherve) [inline README.md:46]: OK for the issue first, here is my suggestion then:

```suggestion
* Open an issue on [jenkins-infra/helpdesk](https://github.com/jenkins-infra/helpdesk) describing the reason why you need an access to the VPN
* Create a new pull request on [jenkins-infra/docker-openvpn](https://github.com/jenkins-infra/docker-openvpn) with a link to the issue previously created
```

File: README.md [hunk 2]
```diff
@@ -40,10 +40,15 @@ Feel free to follow the next action points:
 * Generate your private key and certificate request: `./easyvpn request <your-jenkins-username>`
   Your private key will be generated in `./cert/pki/private`
 
-  > ### This key **must** remain **secret**!
+  > ⚠️ This key **must** remain **secret**! ⚠️
+
+* Open an issue on [jenkins-infra/helpdesk](https://github.com/jenkins-infra/helpdesk) describing the reason why you need an access to the VPN
```
Comment (timja) [inline README.md:45]: this just sounds like double work, can't you just put the reason in the PR?

(PR template could be used)

we get so few VPN requests anyway that we will probably know why someone is requesting it

File: README.md (hunk 1 above)
Comment (dduportal) [reply README.md]: I've moved the "issue" bullet earlier as discussed together

**Comment by dduportal** [comment]
Hello @imonteroperez can I ask you to review (if you have time) this PR to check that every blockers you had today have been addressed on the readme. Many thanks!

If no feedbacks the 12 of April, then we'll proceed to merge (that would mean that you don't have time or not willing to ;) )

**Review by imonteroperez — APPROVED** [review]
LGTM

**Pull request merged**